Makefile

```
@@ -771,6 +767,145 @@ otel-forward: ## Port-forward Grafana (3000)
+PERF_RAMP_RATE ?= 8
+PERF_DURATION ?= 5m
+PERF_HOLD_DURATION ?= 5m
+PERF_TARGET_URL ?= http://localhost:8001/mcp
+PERF_TOOL_PREFIX ?= mock_
+
+.PHONY: perf-build-mock-server
+perf-build-mock-server: ## Build perf mock server image
+	cd tests/perf/mock-server && $(CONTAINER_ENGINE) build $(CONTAINER_ENGINE_EXTRA_FLAGS) -t $(PERF_MOCK_IMG) .
+
+.PHONY: perf-build-k6
+perf-build-k6: ## Build k6 binary with xk6-mcp extension
```

checkmake (0.3.2)

[warning] Target body for "perf-build-k6" exceeds allowed length of 5 lines (7).

(maxbodylength)